Presentation.Web/Models/API/V2/Internal/Request/Organizations/OrganizationGridConfigurationRequestDTO.cs

```diff
@@ -1,14 +1,14 @@
-﻿using Core.DomainModel;
-using System;
+﻿using System;
 using System.Collections.Generic;
+using Core.DomainModel;
 
-namespace Presentation.Web.Models.API.V1
+namespace Presentation.Web.Models.API.V2.Internal.Request.Organizations
 {
     public class OrganizationGridConfigurationRequestDTO
     {
         public Guid OrganizationUuid { get; set; }
```

Comment:
The orgUuid is passed in the URL, same goes for the overviewType. Version is never used.

Why not just the VisibleColumns property, the others aren't used either way.